app/content/carto-user-manual/connections/deleting-a-connection.md

```diff
@@ -1,9 +1,15 @@
 ## Deleting a connection
 
-In the *Connections* section of the Workspace, you can view the list of your current connections. You can access the quick actions menu to manage your connection by clicking on the "three dots" icon in the top-right corner. There are 2 options available: Edit connection and Delete connection.
+In the *Connections* section of the Workspace, you can view the list of your current connections. You can access the quick actions menu to manage your connection by clicking on the "three dots" icon in the top-right corner. There are 3 options available: Edit, Share and Delete.
 
-![Connection quick actions](/img/cloud-native-workspace/connections/the_connections_connection_delete.png)
+![Connection quick actions](/img/cloud-native-workspace/connections/the_connections_connection_deleting.png)
 
 If you click the *Delete* quick action, a dialog will appear allowing you to confirm that you want to delete the selected connection. It includes information about maps related to the existing connection that could potentially be affected by the action. Click the *Yes, delete* button to confirm the changes or click *Cancel* if you don't want the changes to be applied.
 
 ![Connection confirmation deletion](/img/cloud-native-workspace/connections/the_connections_delete_confirmation.png)
+
+Please note that if your connection is shared with your organization, the delete option will not be available and you will first need to make the connection private again. 

```

Comment:
```suggestion
Please note that if your connection is shared with your organization, the delete option will not be available and you will first need to make the connection private again. 

Once is private nobody except you will have access anymore, but you also need to make sure no other users are still using this connection in their maps before fully deleting it. This is in order to avoid deleting other users maps without warning.

```